components/emoji_picker/emoji_picker.jsx

```diff
@@ -17,7 +17,7 @@ import EmojiPickerPreview from './components/emoji_picker_preview';
 
 const CATEGORY_SEARCH_RESULTS = 'searchResults';
 const EMOJI_HEIGHT = 27;
-const EMOJI_CONTAINER_HEIGHT = 244;
+const EMOJI_CONTAINER_HEIGHT = 290;
```

Comment:
This add's some empty space between emoji picker button and gyfcat container which is not the case on pre-release.  

<img width="361" alt="screen shot 2018-07-03 at 11 28 53 am" src="https://user-images.githubusercontent.com/4973621/42201398-4d47f58e-7eb5-11e8-971e-f97910fc0e9f.png">

Reply:
Yup, I noticed that.  Is there a way we can increase the width of the images a bit?

Reply:
Should be fixed now @sudheerDev. Have a look.
![screen shot 2018-07-03 at 5 14 29 pm](https://user-images.githubusercontent.com/11034289/42219212-9afaa23e-7ee4-11e8-9ab2-b0c85b8f3a42.png)

Reply:
@asaadmahmood i meant the space from emoji picker button to the emoji picker has increased.
<img width="358" alt="screen shot 2018-07-03 at 6 05 55 pm" src="https://user-images.githubusercontent.com/4973621/42220182-e1810e9e-7eeb-11e8-9c04-bb97fd5449d9.png">

this height is hard coded here at https://github.com/mattermost/mattermost-webapp/blob/91a518aa363cf913bf1f544f3bbf56f787aadcce/components/gif_picker/components/SearchGrid/SearchGrid.scss#L7

For reference: changing that to `336px` did the trick for me.

Reply:
Oh I see. Fixed @sudheerDev